lms/djangoapps/support/static/support/jsx/success.jsx

```diff
@@ -0,0 +1,49 @@
+/* global gettext */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Success({homepageUrl, dashboardUrl, isLoggedIn}) {
+  let btnText,
+    btnUrl;
+  if (isLoggedIn) {
+    btnText = 'Go to my Dashboard';
+    btnUrl = {dashboardUrl}
```

Comment:
I'm not positive, but I don't think this needs to be wrapped in curly braces. Can you confirm?